### ISSUE TITLE
build error

### ISSUE THREAD

**Issue by hgati** [pipech/erpnext-docker-debian#2]
https://github.com/pipech/erpnext-docker-debian/blob/master/Dockerfile
dockerfile build error.
how can i fix ?

```sh
Cloning into '/tmp/.bench'...

Traceback (most recent call last):
  File "install.py", line 400, in <module>

    install_bench(args)
  File "install.py", line 83, in install_bench
    raise Exception('Please run this script as a non-root user with sudo privileges, but without using sudo or pass --user=USER')
Exception: Please run this script as a non-root user with sudo privileges, but without using sudo or pass --user=USER

Error: ResponseItem.ErrorDetail[code=1,message=The command '/bin/sh -c wget $easyinstallRepo     && python install.py     --without-site     --$benchSetup     --mysql-root-password $mysqlPass      --admin-password $adminPass      --bench-name $benchName      --bench-branch $benchBranch' returned a non-zero code: 1]
Failed to deploy 'erpnext Dockerfile: Dockerfile': The command '/bin/sh -c wget $easyinstallRepo     && python install.py     --without-site     --$benchSetup     --mysql-root-password $mysqlPass      --admin-password $adminPass      --bench-name $benchName      --bench-branch $benchBranch' returned a non-zero code: 1
```

**Comment by pipech** [comment]
Hi,

I planned to look into this in about 2 weeks, a lot has change since i build this Dockerfile, as I said in readme.md frappe has a very rapid development cycle.

In the meantime you should use pre-build image [here](https://hub.docker.com/r/pipech/erpnext-docker-debian/).

**Comment by hgati** [comment]
I trying to build v10.1.18 by `bench update` based on your pre-build v10.0.19 image.
thanks.

**Comment by pipech** [comment]
Dockerfile has been fixed, current version is v10.0.23 😉

**Comment by hgati** [comment]
Great ~~~
